lib/puppet/parser/functions/validate_slength.rb

```diff
@@ -44,7 +44,7 @@ module Puppet::Parser::Functions
     end
 
     if min_length > max_length
-      raise Puppet::ParseError, "validate_slength(): Expected second argument to be larger than third argument"
+      raise Puppet::ParseError, "validate_slength(): Expected second argument to be equal to or larger than third argument"
```

Comment:
The second argument is 'max_length' and the third is 'min_length'.  Or are you suggesting reversing the comparison in L46 for clarity?